app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/state/MessageComposerStateHolder.kt

```diff
@@ -109,6 +109,7 @@ class MessageComposerStateHolder(
     }
 
     fun toReply(message: UIMessage.Regular) {
+        messageCompositionHolder.clearMessage()
```

Comment:
If the flow happens:
`select message to edit -> select another message to reply`

the content of the previous message will be kept in the input field, so here we are clearing the message.